doc/source/train/user-guides/data-loading-preprocessing.rst

```diff
@@ -416,14 +428,34 @@ Random shuffling
 ----------------
 Randomly shuffling data for each epoch can be important for model quality depending on what model you are training.
 
-Ray Data has two approaches to random shuffling:
+Ray Data provides following options to random shuffling, sorted by compute overheads and randomness in acceding order. These options can be used in combination.
```

Comment:
by the way, i added this content in a new page under Ray Data: https://docs.ray.io/en/master/data/shuffling-data.html#types-of-shuffling

we could either link to that page, or omit this section.

Reply:
will link to that page